src/site/content/en/lighthouse-seo/structured-data/index.md

```diff
@@ -1,29 +1,45 @@
 ---
 layout: post
-title: Manually check structured data is valid
+title: Structured data is valid
```

Comment:
Since this is a manual audit I assume there's no `failureTitle` specified for the audit in the source code

Reply:
Yup!